src/flicking.js

```diff
@@ -775,6 +779,41 @@ eg.module("flicking", ["jQuery", eg, window, document, eg.MovableCoord], functio
 		},
 
 		/**
+		 * Set container's height according to each children's height
+		 * @param direction
+     */
+		_setAdaptiveHeight: function(direction) {
+			var MC = eg.MovableCoord;
+			var $panel;
+			var $first;
+			var $children;
+			var height;
+
+			if (direction === MC.DIRECTION_LEFT) {
+				$panel = this.getNextElement();
+			} else if (direction === MC.DIRECTION_RIGHT) {
+				$panel = this.getPrevElement();
+			} else {
+				$panel = this.getElement();
+			}
+
+			$first = $(":first-child", $panel);
+			height = $first.data("height");
+
+			if (!height) {
+				$children = $panel.children();
+
+				// if panel element has multiple elements as child,
+				// create a wrapper div to measure outerHeight correctly
+				if ($children.length > 1) {
+					$first = $children.wrapAll("<div></div>").parent();

```

Comment:
well, letting modified DOM isn't good idea, because you're transforming the node given by user.
That could cause some problems on applying CSS and can give confusion to user. And also maintaining additional node can give increase cost(even isn't a big) to browser handling it.

So, in my perspective after evaluating height, remove it.